src/controllers/confirm.company.controller.ts

```diff
@@ -4,7 +4,7 @@ import { formatForDisplay, getCompanyProfile } from "../services/company.profile
 import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
 import { createConfirmationStatement, getNextMadeUpToDate } from "../services/confirmation.statement.service";
 import { Session } from "@companieshouse/node-session-handler";
-import { FEATURE_FLAG_PRIVATE_SDK_12052021 } from "../utils/properties";
+import { FEATURE_FLAG_PRIVATE_SDK_12052021, FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY } from "../utils/properties";
```

Comment:
feature flag FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY needs the date added to its name

Reply:
feature flag to be updated in separate PR